bigbluebutton-html5/private/config/settings.yml

```diff
@@ -350,23 +350,23 @@ private:
       enabled: false
   minBrowserVersions:
     - browser: chrome
-      version: 59
+      version: 62
     - browser: chromeMobileIOS
       version: Infinity
     - browser: firefox
-      version: 52
+      version: 63
     - browser: firefoxMobile
-      version: 52
+      version: 68
     - browser: edge
-      version: 17
+      version: 72
```

Comment:
Is Edge really reporting as 72?

Reply:
Yes
[Edge versions](https://en.wikipedia.org/wiki/Microsoft_Edge#Chromium_(2019%E2%80%93present))